docs/cep/query-guide/functions/json/tokenize.md

```diff
@@ -33,24 +36,24 @@ CREATE STREAM InputStream (json STRING, path STRING);
 @info(name = 'query1')
 INSERT INTO OutputStream
 SELECT path, jsonElement
-FROM InputStream#json:tokenizeAsObject(json, path);
+FROM InputStream#json:tokenize(json, path);
 ```
 
-In this example, the query named 'query1' processes records from the `InputStream` and extracts JSON elements based on the provided `path` using the `json:tokenizeAsObject(json, path)` function. The resulting `(path, jsonElement)` pairs are inserted into the `OutputStream`.
+In this example, the query named 'query1' processes records from the `InputStream` and extracts JSON elements based on the provided `path` using the `json:tokenize(json, path)` function. The resulting `(path, jsonElement)` pairs are inserted into the `OutputStream`.
 
-For example, if the input `json` is `{name:'John', enrolledSubjects:['Mathematics', 'Physics']}`, and the `path` is passed as `$.enrolledSubjects`, then for both the elements in the selected JSON array, the query generates events as `('$.enrolledSubjects', 'Mathematics')` and `('$.enrolledSubjects', 'Physics')`. For the same input JSON, if the `path` is passed as `$.name`, then it will only produce one event `('$.name', 'John')` as the `path` provided a single JSON element.
+For example, if the input `json` is `{name:'John', address:{country:'CA', city: 'Toronto'}}`, and the `path` is passed as `$.address`, then the query generates an event with the escaped JSON as a string ('$.address', '{\'country\':\'CA\',\'city\':\'Toronto\'}'). For the same input JSON, if the `path` is passed as `$.name`, then it will produce an event without escaping ('$.name', 'John').
```

Comment:
@diana-macrometa We should explain an array example here as that is more useful.. so the earlier example had enrolledSubjects array which is good.. or else have both examples .. but let's not remove an array example

Reply:
```suggestion
For example, if the input `json` is `{name:'John', address:{country:'CA', city: 'Toronto'}, phoneNo: [9876756567, 8778787768]}`, and the `path` is passed as `$.address`, then the query generates an event with the escaped JSON as a string `'$.address', '{\'country\':\'CA\',\'city\':\'Toronto\'}'`. For the same input JSON, if the `path` is passed as `$.name`, then it will produce an event without escaping ('$.name', 'John').
For the same input JSON, if the `path` is passed as `$.phoneNo`, then for both the elements in the selected JSON array, the query generates events with the escaped JSON as a string `{"path":"$.phoneNo","jsonElement":"9876756567"}` and `{"path":"$.phoneNo","jsonElement":"8778787768"}`.
```

Reply:
@diana-macrometa I have added a phoneNo. array to the same example to help explain the array example